OMCompiler/Compiler/FrontEnd/ModelicaBuiltin.mo

```diff
@@ -3132,9 +3132,12 @@ function plot
   input Boolean logX = false "Determines whether or not the horizontal axis is logarithmically scaled.";
   input Boolean logY = false "Determines whether or not the vertical axis is logarithmically scaled.";
   input String xLabel = "time" "This text will be used as the horizontal label in the diagram.";
-  input String yLabel = "" "This text will be used as the vertical label in the diagram.";
+  input String yLabel = "" "This text will be used as the left vertical label in the diagram.";
+  input String yLabelRight = "" "This text will be used as the right vertical label in the diagram.";
   input Real xRange[2] = {0.0,0.0} "Determines the horizontal interval that is visible in the diagram. {0,0} will select a suitable range.";
-  input Real yRange[2] = {0.0,0.0} "Determines the vertical interval that is visible in the diagram. {0,0} will select a suitable range.";
+  input Real yRange[2] = {0.0,0.0} "Determines the left vertical interval that is visible in the diagram. {0,0} will select a suitable range.";
```

Comment:
Add the new fields in the end so the existing scripts using plot does not break.